retries/retries.go

```diff
@@ -11,7 +11,8 @@ import (
 )
 
 type Info[T any] struct {
-	Info    T
+	Info    *T
+	Polling bool
```

Comment:
What does it mean for this field to be false?

It looks like it is set to true everywhere in this PR and never accessed.